EIPS/eip-1.md

```diff
@@ -187,7 +187,14 @@ EIPs may have a `requires` header, indicating the EIP numbers that this EIP depe
 
 ## Linking to External Resources
 
-Links to external resources **SHOULD NOT** be included. External resources may disappear, move, or change unexpectedly.
+External resources are resources that are hosted outside of the EIPs repository. Links to external resources **MUST NOT** be included, except in `Living`, `Meta`, or `Informational` EIPs. External resources may disappear, move, or change unexpectedly. 
+
+The following **MAY** be allowed at editor discretion:
```

Comment:
```suggestion
The following **MAYBE** included without explicit justification:
```
In addition to objection to change SHOULD NOT to MUST NOT
I also *object* to the term `at editor discretion` because it give editors too much power.